stglib/tests/data/11264sg_config.yaml

```diff
@@ -1,5 +1,13 @@
 basefile: 11264sg_raw2
+file_type: .wb
 filename: 11264sg
 LatLonDatum: NAD83
-initial_instrument_height: 0.123
-initial_instrument_height_note: height above seabed
+initial_instrument_height: 1.44
+initial_instrument_height_note: height above seabed in meters
```

Comment:
for instance, here you have height in seabed in meters, but elsewhere you appear to assume it's NAVD88.

Reply:
Ok, I think having it in meters is standard for the config file so I will leave this and fix NAVD88 in the water level utility.